config/kinto.ini

```diff
@@ -0,0 +1,149 @@
+
+
+[server:main]
+use = egg:waitress#main
+host = 0.0.0.0
+port = 8888
+
+
+[app:main]
+use = egg:kinto
+
+#
+# Backends.
+#
+# http://kinto.readthedocs.org/en/latest/configuration/settings.html#storage
+#
+kinto.storage_backend = cliquet.storage.memory
+kinto.storage_url =
+kinto.cache_backend = cliquet.cache.memory
+kinto.cache_url =
+kinto.permission_backend = cliquet.permission.memory
+kinto.permission_url =
+
+#
+# Auth configuration.
+#
+# http://kinto.readthedocs.org/en/latest/configuration/settings.html#authentication
+#
+kinto.userid_hmac_secret = d32712c82034ec704828ff9038c5926fca11463fb92130dd70d8ae7d593294c5
+multiauth.policies = basicauth
+# multiauth.policies = fxa basicauth
+
+# kinto.bucket_create_principals = system.Authenticated
+# kinto.batch_max_requests = 25
+
+#
+# Experimental JSON-schema on collection
+kinto.experimental_collection_schema_validation = true
+
+#
+# Plugins
+#
+kinto.includes = kinto.plugins.default_bucket
+
+#
+# Firefox Accounts configuration.
+#   These are working FxA credentials for localhost:8888
+# kinto.includes  = cliquet_fxa
+# fxa-oauth.client_id = 61c3f791f740c19a
+# fxa-oauth.client_secret = b13739d8a905315314b09fb7b947aaeb62b47c6a4a5efb00c378fdecacd1e95e
+# fxa-oauth.oauth_uri = https://oauth-stable.dev.lcip.org/v1
+# fxa-oauth.requested_scope = profile kinto
+# fxa-oauth.required_scope = kinto
+# fxa-oauth.relier.enabled = true
+# fxa-oauth.webapp.authorized_domains = *
+
+#
+# Client cache headers
+#
+# http://kinto.readthedocs.org/en/latest/configuration/settings.html#client-caching
+#
+# Every bucket objects objects and list
+# kinto.bucket_cache_expires_seconds = 3600
+#
+# Every collection objects and list of every buckets
+# kinto.collection_cache_expires_seconds = 3600
+#
+# Every group objects and list of every buckets
+# kinto.group_cache_expires_seconds = 3600
+#
+# Every records objects and list of every collections
+# kinto.record_cache_expires_seconds = 3600
+#
+# Records in a specific bucket
+# kinto.blog_record_cache_expires_seconds = 3600
+#
+# Records in a specific collection in a specific bucket
+# kinto.blog_article_record_cache_expires_seconds = 3600
+
+#
+# Production settings
+#
+# http://kinto.readthedocs.org/en/latest/configuration/production.html
+#
+# kinto.statsd_url = udp://localhost:8125
+# kinto.statsd_prefix = kinto-prod
+
+# kinto.http_scheme = https
+# kinto.http_host = kinto.services.mozilla.com
+
+# kinto.backoff = 10
+# kinto.retry_after_seconds = 30
+# kinto.eos =
+
+
+# [uwsgi]
+# wsgi-file = app.wsgi
+# enable-threads = true
+# socket = /var/run/uwsgi/kinto.sock
+# chmod-socket = 666
+# processes =  3
+# master = true
+# module = kinto
+# harakiri = 120
+# uid = kinto
+# gid = kinto
+# virtualenv = .venv
+# lazy = true
+# lazy-apps = true
+# single-interpreter = true
+# buffer-size = 65535
+# post-buffering = 65535
+# plugin = python
+
+
+#
+# Logging configuration
+#
+
+# kinto.logging_renderer = cliquet.logs.ClassicLogRenderer
```

Comment:
I'd like all those comments to be removed, they feel useless. Also, maybe this file can be stripped down to only keep the values that are different than the defaults?